### ISSUE TITLE
[STYLE]: Contrast between accent color and background color is quiet low

### ISSUE THREAD

**Issue by AnishDe12020** [tailwindlabs/tailwindcss.com#1020]
The contrast between accent color (light blue) and the background color (white) is quite low. It should be at least 4 and preferably above 7 according to WCAG standards.

![image](https://user-images.githubusercontent.com/63192115/145584112-62db7b75-64e1-4c79-9c3e-ac21043790a8.png)
![image](https://user-images.githubusercontent.com/63192115/145584209-360400e0-4ad7-492d-8a01-32f52c68cc09.png)

**Comment by reinink** [comment]
Hey, thanks for sharing this information. It's always a challenge finding the right balance between creating an appealing design and making sure the site is accessible, which of course is also very important.

We have since launched a dark mode version of the website, so if you're struggling to read this content I'd recommend using that, as it passes the WCAG test  👍

![image](https://user-images.githubusercontent.com/882133/148544649-6d5fec40-cd19-4a5c-b213-2bc44d0c9350.png)

**Comment by AnishDe12020** [comment]
> Hey, thanks for sharing this information. It's always a challenge finding the right balance between creating an appealing design and making sure the site is accessible, which of course is also very important.
> 
> We have since launched a dark mode version of the website, so if you're struggling to read this content I'd recommend using that, as it passes the WCAG test +1
> 
> ![image](https://user-images.githubusercontent.com/882133/148544649-6d5fec40-cd19-4a5c-b213-2bc44d0c9350.png)

Yeah, had made this issue before the dark mode launched. It looks really good and is readable now, thanks a lot :)

**Comment by Myndex** [comment]
> _Worth adding it also fails the newer Advanced Perceptual Contrast Algorithm (APCA)_
> ![image](https://user-images.githubusercontent.com/32498324/146024534-9bc9de69-de16-4833-b1c6-2924f4653c22.png)

Hi Luke @lukewarlow 

Can I ask what contrast checker that is? APCA does not report 100 for this pair but -58 for white text on that blue. That doesn't even seem correct for the 2020 version of APCA... ???

Thank you for any information you can provide.


Hi Jonathan @reinink I have a bit to add:

> _We have since launched a dark mode version of the website, so if you're struggling to read this content I'd recommend using that, **as it passes the WCAG test**_ 

If I can mention a little advisory, WCAG 2 contrast becomes essentially meaningless for dark colors — when both colors are darker than about #999, then suddenly "anything seems to pass". 

If you are working with dark mode, I recommend either the latest version of APCA, or if you need to be backwards compatible with WCAG 2 we released Bridge-PCA — it's compatible with WCAG 2 but using APCA technology.

I have an article that discusses (and demonstrates) how WCAG 2 incorrectly passes dark colors. This link skips the paywall, so it's free to view: https://atangledwebweweave.com/whats-red-black-also-not-read-573b9c0a97ed

### Here's a side by side example:
![ColumnCompareAll400](https://user-images.githubusercontent.com/42009457/148699911-10b3232d-c2f4-426a-a432-9ee702c6f3e4.png)


While the WCAG 2 columns become unreadable for the same contrast level as colors get darker, APCA maintain readability across the visible range.

Please feel free to hit me up with any questions in this regard.


Thank you

Andy


_Andrew Somers_     
_Senior Color Scientist_     
_Myndex Research_     
_Inventor of APCA_

**Comment by lukewarlow** [comment]
It's the experimental checker built into chrome dev tools. Believe it's an opt-in feature for now in the experimental menu.

**Comment by Myndex** [comment]
> It's the experimental checker built into chrome dev tools, so it's probably rough around the edges and ofc it could be user error on my part that did the check wrong.
> 
> Believe it's an opt-in feature for now in the experimental menu.

Hi @lukewarlow   Thank you, I'll file a bug report — is that the latest version? And what OS please so I can add that in?

Thank you!!!!

Andy

**Comment by lukewarlow** [comment]
It would have been whatever version of chrome was in the Dev release channel at the time of the comment unfortunately I'm not able check what it would have been 100%. And I'm on macOS 12. 

I can double check later today if it still gives the same value.

**Comment by reinink** [comment]
@Myndex Hey, bit of a delayed response here, but I just want to say thank you for sharing this information. Really appreciate it!!
